data/items.ts

```diff
@@ -67,6 +67,9 @@ export const Items: {[itemid: string]: ItemData} = {
 			basePower: 30,
 		},
 		onAfterBoost(boost, target, source, effect) {
+			if ((target.boosts['atk'] === -6) || (target.boosts['spe'] === 6)) return;
+			// check contrary users
+			if (target.hasAbility('Contrary') && (target.boosts['atk'] === 6)) return;
```

Comment:
I don't think this is right because `target` has already been bosted at this point so this fails when the target was boosted to 6. Try checking `boost` to see whether there was actually a stat change inflicted.

Reply:
It passes all 7 tests

Reply:
I'm pretty sure urkerab meant check what `boost` was in this function as part of these checks, not during the boosts.